index.html

```diff
@@ -0,0 +1,449 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="icon" href="./favicon.ico" />
+    <title>Aziz Mejri - Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <h1 id="logo">Aziz Mejri</h1>
+      <div class="menu-bars">
+        <i class="fas fa-bars open-btn"></i>
+        <i class="fas fa-times close-btn"></i>
+      </div>
+
+      <nav id="desktop-nav">
+        <ul>
+          <li><a class="nav-link" href="#projects">Portoflio</a></li>
+          <li><a class="nav-link" href="#about-me">About</a></li>
+          <li>
+            <a class="nav-link" href="#about-me"
```

Comment:
![comparison11](https://user-images.githubusercontent.com/49184579/175209419-96e8ddf3-805a-4c4f-ae88-55a2e3e2a42b.png)

The navigation items are not visible on the desktop-size screen. Also please follow the Figma Design as closely as possible.